src/Sulu/Bundle/SecurityBundle/Entity/BaseRole.php

```diff
@@ -62,6 +62,7 @@
 
     public function __construct()
     {
+        parent::__construct('');
```

Comment:
missing `parent constructor call` will set $role attribute to empty. See https://github.com/symfony/security/blob/v3.3.13/Core/Role/Role.php#L28